### DISCUSSION TITLE
No update datebase

### DISCUSSION THREAD

**Discussion by Jan-Kruis** [postfixadmin/postfixadmin#710] [General]
Hi,

I recently ran into a problem.

I was unable to adjust the holiday setting.
I didn't get an error message from Postfixadmin after I made changes via public/vacation.php, but they were not updated in the database.
After investigation it turned out that the 'reply type' field in the 'vacation' table still existed. this field was created with update 1345 at the time. Although update 1345 was canceled in update 1610 , the field created in update 1345 was never removed by a later update. Deleting the field manually solved the problem. Perhaps there should be an adjustment in the update.php to remove fields that are no longer used.

This brings me to my second question: is the field 'cache' in table 'vacation' still used (I don't think so)? Maybe there are several unused fields and they could be removed with a change in update.php. I would love to hear from you.

Regards,
Jan Kruis

**Comment by DavidGoodwin** [comment]
* vacation.cache -> I think it's no longer used; See model/VacationHandler.php which has a relevant comment - https://github.com/postfixadmin/postfixadmin/blob/18b3638787e86526536a64ce60e00b22e0bb109b/model/VacationHandler.php#L48
 * stale fields - probably worth removing, but there's not much to be gained by doing it ...
